plugins/de.cau.cs.kieler.klighd.ui/src/de/cau/cs/kieler/klighd/ui/printing/dialog/RangeBlock.java

```diff
@@ -106,36 +109,46 @@ public static Group createContents(final Composite parent, final DataBindingCont
         // the update of 'rangeValue' is than caused by the de-selection of 'allValue' leading to
         //  a re-computation of rangValue by means of the above mentioned listeners (crazy shit ...);
         //  the than calculated result is used below for enabling the "from" & "to" text fields.
-        final IObservableValue rangeValue = new ComputedValue(realm) {
+        final IObservableValue rangeValue = new ComputedValue<>(realm) {
```

Comment:
For me this replaces the warning with an error: "'<>' cannot be used with anonymous classes"

Reply:
Your fix did not fix this error for me, replacing the <> with <Boolean> as well fixes it for me.

Reply:
Oh, I thought I had fixed this but apparently I somehow missed this in 145ba1a33fc32914952fc95d94f95a533f2871f5

Reply:
This should now be fixed in 75cc039e7814e12333012c092d4401be538072ad